Makefile.am

```diff
@@ -19,6 +19,10 @@ BUILT_SOURCES = \
 	proto/ct.pb.cc \
 	proto/ct.pb.h
 
+CLEANFILES = \
```

Comment:
This is an Automake variable: https://www.gnu.org/software/automake/manual/automake.html#Clean